tests/create/scaffold/scaffold_e2e_test.go

```diff
@@ -0,0 +1,396 @@
+package scaffold_test
```

Comment:
I like that you are already providing more than unit tests!
These are not really E2E tests because we are using the compiled ctl bin to execute.
But you can call them integration tests let's say.
Can you move these into `tests/integration/create/scaffold/` please!